conf/broker.conf

```diff
@@ -800,6 +800,13 @@ managedLedgerDefaultWriteQuorum=2
 # Number of guaranteed copies (acks to wait before write is complete)
 managedLedgerDefaultAckQuorum=2
 
+# with OpportunisticStriping=true the ensembleSize is adapted automatically to writeQuorum
+# in case of lack of enough bookies
+#bookkeeper.opportunisticStriping=false
```

Comment:
Can we avoid using "."? Currently, we use system environment variables for loading and applying config in Kubernetes. Using "." will make things become challenging. Can we use `bookkeeper_` instead?

Reply:
yes, good idea @sijie .
I have updated the patch